src/test/java/com/epam/steps/common/LoginSteps.java

```diff
@@ -68,9 +71,9 @@ public void fillEmailAndPassword(String email, String password) {
 
     @Then("Check error message")
     public void checkErrorMessage() {
-        assertThat(loginPage
-                .getErrorMessage())
-                .isEqualTo("Incorrect email and/or password");
+        assertThat(loginPage.getErrorMessage())
+                .withFailMessage("Error message for incorrect login/or password was different from expected one.")
+                .isEqualTo(INCORRECT_LOGIN_OR_PASSWORD.getErrorMessage());
     }
 
     @And("Fill in all required fields")
```

Comment:
I think  the methods which are implemented below this line besides signInAsAdminWithGeneratedPassword() shouldn't be in this class, We need move them to CreateUserSteps class

Reply:
done